js/style/style_function.js

```diff
@@ -8,7 +8,9 @@ exports.interpolated = function(parameters) {
         return inner(globalProperties && globalProperties.zoom, featureProperties || {});
     };
     outer.isFeatureConstant = inner.isFeatureConstant;
-    outer.isGlobalConstant = inner.isGlobalConstant;
+    outer.isZoomConstant = inner.isZoomConstant;
+    outer.getStopZoomLevels = inner.getStopZoomLevels;
```

Comment:
I didn't consider parsing them out in mapbox-gl-js. Putting the function as member of the real function isn't great. Having it in mapbox-gl-function means -js doesn't need to know about the syntax of the functions though. Do you have any ideas on how this could be cleaned up?